lib/google_drive/spreadsheet.rb

```diff
@@ -131,5 +131,35 @@ def batch_update(requests)
         @session.sheets_service.batch_update_spreadsheet(id, batch_request)
       batch_response.replies
     end
+
+    # Append values to a spreadsheet by first searching for a data table at a range,
+    # then appending the specified values at the end of this data table.
+    #
+    # +range+ The A1 notation of a range to search for a logical table of data. 
+    # Values will be appended after the last row of the table.
+    # +values+ Array (rows) of Array (columns) of values to append to the spreadsheet.
+    # +override_params+ allows you to control how the values will be inserted.
+    #   By default, the values will be interpreted as if typed by a user, 
+    #   and will add new rows instead of ovewriting existing ones.
```

Comment:
```suggestion
    #   By default, the values will be interpreted as if typed by a user, 
    #   and will add new rows instead of ovewriting existing ones.
    #   So default value is `{ value_input_option: 'USER_ENTERED', insert_data_option: 'INSERT_ROWS' }`
    #   See https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values/append#query-parameters for more information
```